src/pages/Checkmark.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import './Checkmark.css';
+
+function Checkmark() {
+  return <input type="checkbox"></input>;
```

Comment:
Non-blocking: The checkbox could also be written as: `<input type="checkbox" />`